src/components/header.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from "gatsby"
+import PropTypes from "prop-types"
+import React from "react"
+
+const Header = ({ siteTitle }) => (
+  <header
+    style={{
```

Comment:
Down below we'll see an example of an entire CSS file. This is an example of inline CSS.